include/ListView/ListViewController.php

```diff
@@ -440,6 +467,9 @@ function getListViewEntries($focus, $module,$result,$navigationInfo,$skipActions
 						$value = '--';
 					}
 				} elseif($field->getFieldDataType() == 'owner') {
+                                        if($fieldName!='assigned_user_id' && strstr($fieldName,".assigned_user_id"))
```

Comment:
Since you have to modify the file to fix the above comment I would appreciate you added the block delimeters to this "if":

```
if($fieldName!='assigned_user_id' && strstr($fieldName,".assigned_user_id")) {
$value = textlen...
} else {
....
}
```

Thanks